Gaijinizer patch/Enemies.txt

```diff
@@ -2906,14 +2910,12 @@ M.O.W生物兵器　■■-■■■■（ロストナンバー）『双子』
 彼女たちの物言いや、過去にも変わらぬ姿で現れていることから
 『時空を超越する能力』が備わっていると推測される。
 > CONTEXT: Enemies/619/BookDescription/2
-M.O.W. Biological weapons ■ ■ ■ ■ ■ ■ ■ ■ (lost number)
-"Twins".
-It is the cause of Mukuro's birth and the cause of the
-ancient disaster.
-Because they have appeared in the same manner as the
-women's stories and the past
-It is speculated that it has the ability to transcend
-space-time.#MTLed
+M.O.W. Biological Weapons ■■-■■■■ (lost number) "Twins".
+The cause of Mukuro's birth and the culprits behind the
```

Comment:
Do we want to localize "lost number"?

Reply:
Maybe something more machine-like such as "number not found"?

Reply:
Yeah... talked about it in server, and this should be good:
```
M.O.W. Biological Weapons ■■-■■■■[ID NOT FOUND] "Twins".
The cause of Mukuro's birth and the culprits behind
the ancient calamity. Due to the way they speak and
the fact that their appearance is the same as it was
in the past, it is speculated that they possess the
ability to transcend time and space.#Strate
```